2.6/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM centos:centos7
+MAINTAINER  Jakub Hadvig <jhadvig@redhat.com>
```

Comment:
Please check how the `MAINTAINER`s are handled in https://github.com/openshift/sti-ruby/tree/master/2.2 and do it pls accordingly